atomic_reactor/inner.py

```diff
@@ -394,6 +391,9 @@ def build_docker_image(self):
                 raise
 
             return self.build_result
+        except Exception as ex:
+            logger.debug("caught exception (%r) so running exit plugins", ex)
```

Comment:
That's right. If an exit plugin later raises its own exception, we'll never find out what the original problem was.